Project.toml

```
@@ -62,8 +63,8 @@ NearestNeighbors = "0.2 - 0.4"
 Parameters = "0.9 - 0.12"
 SpecialFunctions = "1.0, 2"
 StaticArrays = "1"
-WriteVTK = "1"
 WhereTheWaterFlows = "0.10"
+WriteVTK = "1"
 julia = "1.9"
```

Don't we need to specify newer versions on the project as well?
```suggestion
julia = "1.9, 1.10"
```

---

no, 1.9 is the lower bound. We only need to put commas if it is a breaking release. 
For packages that have version >=1.0, breaking releases are 1.0, 2.0 etc. For packages <1.0, a breaking release is 0.1, 0.2, 0.3, etc.